testRunner.js

```diff
@@ -0,0 +1,54 @@
+import fs from 'fs';
+import glob from 'glob';
+import { execSync } from 'child_process';
+
+// get directories in src
+const directories = fs
+  .readdirSync('./src/restapi', { withFileTypes: true })
+  .filter((dirent) => dirent.isDirectory())
+  .map((dirent) => dirent.name);
+
+/*
+  For each directory, run vitest sequentially.
+
+  This is done to make sure that the tests are isolated from which other.
+  
+  Using no-threads parameter removes the per-module isolation as well, but 
+  not using the no-threads parameter causes tests to fail beccause there
```

Comment:
There appear to be some issues related to 'resetFixtures.' The tests need to be isolated from each other because they can interfere with each other's mutations. If this is not done, some of the tests may encounter timeout errors in a random order.

Reply:
Using no-threads parameter removes the per-module isolation as well, but not using the no-threads parameter causes tests to fail because is no isolation at the test DB level and different tests can interefere with each other's mutations.